website/docs/docs/build/metrics-overview.md

```diff
@@ -48,6 +48,27 @@ This page explains the different supported metric types you can add to your dbt
 - [Ratio](#ratio-metrics) — Create a ratio out of two measures. 
 -->
 
+### Conversion metrics <Lifecycle status='new'/>
```

Comment:
adding this bc it's a new feature for users and falls within the 'lifecycle' scope. will remove next month (reminder set)